### DISCUSSION TITLE
whether to support selecting other fork flutter sdk versions

### DISCUSSION THREAD

**Discussion by shAdow-XJY** [codemagic-ci-cd/codemagic-docs#1678] [Q&A] [answered]
I would like to ask, emmm.....
When code magic is applied in build flutter, whether to support selecting other fork flutter sdk versions.
For example, from flutter official fork, and modified flutter sdk，and I provide the link

**Comment by dtrdic** [comment, marked as answer]
hey @shAdow-XJY, i'm not sure if i understood you correctly. Are you asking if CodeMagic supports the ability to choose and use different forked versions of the Flutter SDK during the build process?

if yes, you can use FVM (Flutter Version Management) to manage and switch between different versions of the Flutter SDK, including forked versions.

You can check it here on how it's done https://docs.codemagic.io/yaml-quick-start/building-a-flutter-app/#setting-the-flutter-version

**Reply by shAdow-XJY** [reply]
hello, @dtrdic .
yeah, that's what I mean.
I wonder if CodeMagic supports the ability to choose and use different forked versions of the Flutter SDK during the build process.
